doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1643,6 +1643,44 @@ For example:
 One can always specify the parameter if it is not uniform using the
 usual implicit arguments disambiguation syntax.
 
+The syntax is also supported in internal binders. For instance, in the
+following kinds of expressions, the type of each declaration present
+in :token:`binders` can be bracketed to mark the declaration as
+implicit:
+:n:`fun (@ident:forall @binders, @type) => @term`,
+:n:`forall (@ident:forall @binders, @type), @type`,
+:n:`let @ident @binders := @term in @term`,
+:n:`fix @ident @binders := @term in @term` and
+:n:`cofix @ident @binders := @term in @term`.
+Here is an example:
+
+.. coqtop:: all
+
+   Axiom Ax :
+     forall (f:forall {A} (a:A), A * A),
+     let g {A} (x y:A) := (x,y) in
+     f 0 = g 0 0.
+
+.. warn:: Ignoring implicit binder declaration in unexpected position
+
+   This is triggered when setting an argument implicit in an
+   expression which does not correspond to the type of an assumption
+   or to the body of a definition. Here is an example:
+
+.. coqtop:: all
```

Comment:
You should add `warn` to the options so that the build knows that a warning is expected here. It would be better also if you could indent this whole block to be "inside" the `.. warn::` block.

Reply:
OK, I tried to decipher your first sentence and concluded by browsing similar rst code that I should have written `.. coqtop:: all warn` (sorry for my ignorance). I indented and added the missing period. Hopefully, this answers your request, correct?